reflex/state.py

```diff
@@ -1619,7 +1622,22 @@ def _clean(self):
         self.dirty_vars = set()
         self.dirty_substates = set()
 
-    def get_value(self, key: str) -> Any:
+    def get_value(self, value: Any) -> Any:
+        """Get the value of a field (without proxying).
+
+        The returned value will NOT track dirty state updates.
+
+        Args:
+            value: The value of the field.
+
+        Returns:
+            The value of the field.
+        """
+        if isinstance(value, MutableProxy):
+            return super().get_value(value.__wrapped__)
+        return super().get_value(value)
+
+    def _get_value(self, value: Any, **kwargs) -> Any:
```

Comment:
why a new function? is this just WiP artifacts?

Reply:
`_get_value` is there to allow unwrapping the `MutableProxy`s in pydantics serialization. There should be a better way to do this.
Funny sidenote: While `_get_value` is deprecated by pydantic, they somehow seem to rely on it for serialization.